ci-operator/config/open-cluster-management/backplane-operator/open-cluster-management-backplane-operator-main.yaml

```diff
@@ -0,0 +1,44 @@
+base_images:
+  builder-go1.16-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+  open-cluster-management_builder_go1.15-linux-amd64:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.15-linux-amd64
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+images:
+- dockerfile_path: Dockerfile.prow
+  inputs:
+    open-cluster-management_builder_go1.15-linux-amd64:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:go1.15-linux-amd64
+  to: backplane-operator
+promotion:
+  name: "1.0"
+  namespace: open-cluster-management
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
```

Comment:
You need to add the following tests to your main branch:
- sonar-pre-submit
- sonar-post-submit
- e2e-cluster
- e2e-kind
- (optionally) premerge-image-mirror

Your actual testing will be done in the main branch while you're developing for 1.0, not in the backplane-1.0 branch.

When you cut to the next branch (say 2.0), main will do change to do testing for 2.0, but you'll still need the tests in the backplane-1.0 branch to test any z-stream releases.